.github/workflows/build_conda_macos.yml

```
@@ -85,6 +89,7 @@ jobs:
         working-directory: ${{ inputs.repository }}
         run: |
           source "${BUILD_ENV_FILE}"
+          CONDA_EXTRA_BUILD_CONSTRAINT="- mkl<=2021.2.0"
           ${CONDA_RUN} conda build \
             -c defaults \
             -c "pytorch-${CHANNEL}" \
```

Perhaps this minor change in how we set env vars? This worked for me in a different setting so worth a shot.
```suggestion
        working-directory: ${{ inputs.repository }}
        env:
          CONDA_EXTRA_BUILD_CONSTRAINT: "- mkl<=2021.2.0"
        run: |
          source "${BUILD_ENV_FILE}"
          ${CONDA_RUN} conda build \
            -c defaults \
            -c "pytorch-${CHANNEL}" \
```